apps/frontend/src/app/5_pages/StakePage/components/PersonalStakingStatistics/PersonalStakingStatistics.tsx

```diff
@@ -73,16 +73,6 @@ export const PersonalStakingStatistics = () => {
                         {fromWei(votingPower)} {VP}
                       </b>
                     </div>
-                    <div>
-                      {t(
-                        translations.stakePage.personalStatistics
```

Comment:
Delete the translation as well.

Reply:
totally removed tooltip as we discussed, it's no longer needed since we only have 1 item in the breakdown